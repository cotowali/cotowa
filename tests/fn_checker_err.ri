fn f(i int) string {
  return i
}
