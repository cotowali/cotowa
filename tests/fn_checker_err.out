fn_checker_err.ri 2,11: mismatched types: `fn (int) string` (expected) and `int` (found)
  > i
    ^
